halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/SpringServiceSettings.java

```diff
@@ -30,29 +27,14 @@
 @EqualsAndHashCode(callSuper = true)
 @Data
 abstract public class SpringServiceSettings extends ServiceSettings {
-  private static final String SPRING_PROFILES_ACTIVE_KEY = "SPRING_PROFILES_ACTIVE";
-
   protected void setProfiles(List<String> profiles) {
     if (profiles == null || profiles.isEmpty()) {
       return;
     }
 
+    String key = "SPRING_PROFILES_ACTIVE";
```

Comment:
Local variable necessary?

Reply:
Not necessary, but I reverted this file back to what it is in master. I can refactor out the key if it's cleaner.